Bottom: e5ab7140e584fe77dd1b9312643bf7bf89141cfa
Top:    2209669edf35b42156d008a167fd733c633ff77a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 12:11:43 +0000

Refresh of lesstif-silence-set-but-unused-0

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 4bd5391..cb3b01e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -1360,7 +1360,6 @@ static void
 work_area_input (Widget w, XtPointer v, XEvent * e, Boolean * ctd)
 {
   static int pressed_button = 0;
-  static int ignore_release = 0;
 
   show_crosshair (0);
   switch (e->type)
@@ -1382,11 +1381,7 @@ work_area_input (Widget w, XtPointer v, XEvent * e, Boolean * ctd)
           return;
         /*printf("click %d\n", e->xbutton.button); */
         if (lesstif_button_event (w, e))
-	{
-	  ignore_release = 1;
-	  return;
-	}
-        ignore_release = 0;
+          return;
 
         notify_crosshair_change (false);
         pressed_button = e->xbutton.button;
